yanfd.toml.sample

```diff
@@ -133,3 +133,15 @@
 
     # Enables or disables auto prefix propagation
     auto_prefix_propagation = false
+  
+  [tables.fib]
+    # Selects the algorithm used to implement the FIB
+    # Allowed options: nametree, hashtable
+    # Default: nametree
+    algorithm = nametree
```

Comment:
When I run YaNFD with this updated config file, I get the following error:

```
2022/05/10 23:57:26 fatal [Config] Unable to load configuration file: (137, 17): no value can start with h
```

Do you know how to fix this? Recompiling YaNFD doesn't fix it for some reason.

Reply:
Maybe try `"nametree"`

Reply:
That fixed it, thanks!